amgut/test/test_dbpatch_source.py

```diff
@@ -0,0 +1,104 @@
+from unittest import TestCase, main
```

Comment:
I figure we need to manually change those numbers to a new scrubbed version, because our DB will grow -> i.e. very sensitive to DB updates